_sass/lib/src/_mixins.scss

```diff
@@ -1,3 +1,5 @@
+@use 'sass:math'
```

Comment:
i need a semicolon to make it work

Reply:
Ok let me add a semicolon. Thanks for checking it.